src/Controls/src/Build.Tasks/nuget/buildTransitive/net6.0-maccatalyst13.1/Microsoft.Maui.Controls.DefaultItems.targets

```diff
@@ -0,0 +1,12 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This is a new file, so just checking where this is getting imported? I would think this could be done in the `Microsoft.Maui.Controls.MacCatalyst.targets` file.